packages/keycloak-user-management/src/components/CreateEditUser/tests/index.test.tsx

```diff
@@ -291,70 +371,13 @@ describe('components/CreateEditUser', () => {
       wrapper.update();
     });
 
-    expect(wrapper.find('CreateEditUser').prop('keycloakUser')).toEqual(fixtures.keycloakUser);
+    expect(wrapper.find('CreateEditUser').prop('keycloakUser')).toEqual(keycloakUser);
     expect(wrapper.find('CreateEditUser').prop('accessToken')).toEqual('bamboocha');
     expect(wrapper.find('CreateEditUser').prop('extraData')).toEqual({
       api_token: 'hunter2',
       oAuth2Data: { access_token: 'bamboocha', state: 'abcde' },
     });
-    expect(mockSelector).toHaveBeenCalled();
-    wrapper.unmount();
-  });
-
-  it('initializes form fields with user data if page is refreshed', async () => {
```

Comment:
Was there a need to remove this test case?

Reply:
yes i think you are me or maybe kip  discussed about this specific one being failed cuz of multiple calls